sources-docs.js

```diff
@@ -3,18 +3,18 @@ module.exports = [
         resolve: `gatsby-source-git`,
         options: {
           name: `docs--4`,
-          remote: `https://github.com/silverstripe/silverstripe-framework.git`,
-          branch: `4`,
-          patterns: `docs/en/**`
+          remote: `https://github.com/silverstripe/developer-docs.git`,
+          branch: `4.11`,
```

Comment:
Will this get updated each time we do a release?

Reply:
It will once https://github.com/silverstripe/doc.silverstripe.org/issues/175 has been done.